bokeh/models/annotations.py

```diff
@@ -333,7 +346,11 @@ class Legend(Annotation):
 
     where each tuple is of the form: *(label, renderers)*.
 
-    """).accepts(List(Tuple(String, List(Instance(GlyphRenderer)))), lambda items: [LegendItem(label=item[0], renderers=item[1]) for item in items])
+    """).accepts(List(Tuple(String, List(Instance(GlyphRenderer)))), lambda items: [LegendItem(label=item[0], 
+                                                                                               renderers=item[1], 
+                                                                                               index=Int(default=0), 
+                                                                                               size=Int(default=20)
```

Comment:
These should just be `index=0` and `size=20` the property types are only used on class definitions

Reply:
That said, do we need to specify the index here?

Reply:
I actually don't remember why I added index and size. Must have been testing something. Removing!